build.yaml

```diff
@@ -315,16 +351,21 @@ common_cli:
 
   build_package: &build_package  |
     # and build a rpm/deb package using docker
-    parse_logs \$venv/bin/python admin/build-package --destination-path repo --distribution \${DISTRIBUTION_NAME} /flocker/dist/Flocker-\${FLOCKER_VERSION}.tar.gz
+    parse_logs \$venv/bin/python admin/build-package \
+    --destination-path repo \
+    --distribution \${DISTRIBUTION_NAME} \
+    /flocker/dist/Flocker-\${FLOCKER_VERSION}.tar.gz
 
   build_repo_metadata: &build_repo_metadata |
     # the acceptance tests look for a package in a yum repository,
     # we provide one by starting a webserver and pointing the tests
     # to look over there
-    sudo rm -rf /usr/share/nginx/html/results/omnibus/\${TRIGGERED_BRANCH}/\${DISTRIBUTION_NAME}
-    sudo mkdir -p /usr/share/nginx/html/results/omnibus/\${TRIGGERED_BRANCH}/\${DISTRIBUTION_NAME}
-    sudo cp repo/* /usr/share/nginx/html/results/omnibus/\${TRIGGERED_BRANCH}/\${DISTRIBUTION_NAME}
-    cd /usr/share/nginx/html/results/omnibus/\${TRIGGERED_BRANCH}/\${DISTRIBUTION_NAME}
+    REPO_PATH=/results/omnibus/\${TRIGGERED_BRANCH}/\${DISTRIBUTION_NAME}
+    DOC_ROOT=/usr/share/nginx/html
+    sudo rm -rf \${DOC_ROOT}/\${REPO_PATH}
```

Comment:
well, it doesn't have to be shell,
It can be anything, even go https://wiki.ubuntu.com/gorun.

I assume this will eventually get ported to python